fms_core/models.py

```diff
@@ -47,7 +39,7 @@ class Sample(models.Model):
     experimental_group = JSONField(blank=True, null=True)
     # only three types
     # redundant ?
-    container_kind = models.CharField(choices=CONTAINER_KIND)
+    container_kind = models.CharField(max_length=20, choices=SAMPLE_CONTAINER_KIND_CHOICES)
```

Comment:
This does feel redundant - we can check the spreadsheet value against the barcode's true value for validation, but we shouldn't store it